tests/core/tests/test_admin_integration.py

```diff
@@ -49,8 +49,11 @@ def test_import_export_template(self):
         self.assertEqual(response.status_code, 200)
         self.assertTemplateUsed(response,
                 'admin/import_export/change_list_import_export.html')
+        self.assertTemplateUsed(response, 'admin/change_list.html')
+        self.assertTemplateUsed(response, 'core/admin/change_list.html')
         self.assertContains(response, _('Import'))
         self.assertContains(response, _('Export'))
+        self.assertContains(response, 'Custom change list item')
```

Comment:
I might be missing something but I don't think the functionality of overriding base_change_list_template is tested, is that something that would be useful to test to ensure future compatibility or am I misunderstanding how that should be used? Thanks!

Reply:
👍